### DISCUSSION TITLE
microphone difference

### DISCUSSION THREAD

**Discussion by nsdolic** [Ant-Brain/EfficientWord-Net#46] [General]
Hi all - I am recording samples for trigger word training on raspberry pi4 using usb mic. (cheap one). Recording about 10 samples and running "eff_word_net.generate_reference" works just fine. 
However, if I use airpods to record samples, I can't make it work. Any ideas what might be different? I did manually check each sample and could not detect anything obvious.

Thank you all!!!

**Comment by TheSeriousProgrammer** [comment]
Its possible that the airpods may not be connected well with your system, are you using a mac or a raspberry pi to test this out?

**Reply by TheSeriousProgrammer** [reply]
I'd suggest trying to record a few sample audio files through some simple linux commands such as "rec" and "arec", these will tell you if your airpods are discoverable for the system

**Comment by nsdolic** [comment]
Thank you for your response. 

I actually made it work by selecting codec mSBC instead of codec CVSD. Not sure why though.
